examples/default/Makefile

```diff
@@ -33,6 +33,7 @@ USEMODULE += shell_commands
 USEMODULE += ps
 # include and auto-initialize all available sensors
 USEMODULE += saul_default
+USEMODULE += ina219
```

Comment:
huh?
I don't think this belongs here.

Reply:
Nice catch!

Reply:
Fixed. Thanks for spotting this. That would have been embaressing again :-/